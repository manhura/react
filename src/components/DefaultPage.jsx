import React from 'react';

const DefaultPage = () => (
  <div>
    4041
  </div>
);

export default DefaultPage;