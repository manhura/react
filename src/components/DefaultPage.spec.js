import React from 'react';

import renderer from 'react-test-renderer';

import DefaultPage from './DefaultPage';

it('renders correctly', () => {
  const tree = renderer
    .create(<DefaultPage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});