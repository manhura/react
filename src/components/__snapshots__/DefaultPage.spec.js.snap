// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  4041
</div>
`;
